Example/InstanceID/Tests/FIRInstanceIDTest.m

```diff
@@ -156,6 +156,8 @@ - (void)testSharedInstance {
   // The shared instance relies on the default app being configured. Configure it.
   FIROptions *options = [[FIROptions alloc] initWithGoogleAppID:kGoogleAppID
                                                     GCMSenderID:kGCMSenderID];
+  options.APIKey = @"api-key";
```

Comment:
out of curiosity: what does the @ symbol do?

Reply:
`"a"` is a C string, `@"b"` is [NSString](https://developer.apple.com/documentation/foundation/nsstring?language=objc)

Reply:
cool, thank you!
NS is a bit of a weird acronym for me as a German... 
but I looked it up, NS stands for NextStep....